acurl/tests/test_httpbin.py

```diff
@@ -39,6 +39,19 @@ async def test_session_cookies(httpbin):
     assert len(cookie_list) == 0
 
 
+@pytest.mark.asyncio
+async def test_session_cookies_sent_on_subsequent_request(httpbin):
+    s = session()
+    await s.get(httpbin.url + "/cookies/set?name=value")
+    cookie_list = s.get_cookie_list()
+    assert len(cookie_list) == 1
+    assert cookie_list[0].name == "name"
```

Comment:
These assertion duplicate what's in `test_session_cookies` so I'd tend to think they don't belong here.  Otherwise this lgtm.

Reply:
Good point, removed these assertions